### ISSUE TITLE
A convenient way to switch scales mode for pads on and off

### ISSUE THREAD

**Issue by orbitfold** [polyend/TrackerIssues#1235]
## Hi, we're excited to hear your idea!

Before you start, did you check whether your feature was already requested?
Searching might save you quite a bit of time here.

### A new feature

I really like the possibility to set different scales for the pads. However I think it would be useful to be able to quickly switch it back and forth from "c chromatic". Now I have to go to the settings for that. Mainly this is to easily be able to switch from "drum" to "melody/harmony" modes.

**Comment by mwmwmw** [comment]
This and being able to transpose the pads without having to go into the config.

Shift Note Up Arrow / Down Arrow for transpose (+/- octave?)
Shift Note Left / Right for scale mode (cycles through?)

**Comment by antf4rm** [comment]
Yes! Navigating to the Config menu for keyboard changes is a huge flow killer.
